components/elm/src/biogeochem/EcosystemBalanceCheckMod.F90

```diff
@@ -966,7 +975,9 @@ subroutine GridCBalanceCheck(bounds, col_cs, col_cf, grc_cs, grc_cf)
 
          if (.not. use_fates) then 
            grc_cinputs(g) = grc_gpp(g) + grc_dwt_seedc_to_leaf(g) + grc_dwt_seedc_to_deadstem(g)
-         end if 
+         else
+           grc_cinputs(g) = grc_cinputs(g) + grc_hrv_deadstemc_to_prod10c(g) + grc_hrv_deadstemc_to_prod100c(g)
```

Comment:
@ckoven is there a fates or e3sm issue that this fixes?

Reply:
Per discussion with @ckoven there is not a prexisting issue.  It addresses issues that Charlie was seeing specific to his branch work.